java/dynamodb-tools/src/main/java/sleeper/dynamodb/tools/DynamoDBUtils.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package sleeper.compaction.status;
+package sleeper.dynamodb.tools;
```

Comment:
The time to live configuration here is specific to the status store code. We could extract that out as a separate function that's called after initialiseTable?